namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-temporary-accommodation-dev/resources/rds.tf

```diff
@@ -0,0 +1,81 @@
+module "rds" {
+  source                       = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.16.11"
+  cluster_name                 = var.cluster_name
+  team_name                    = var.team_name
+  business-unit                = var.business_unit
+  application                  = var.application
+  is-production                = var.is_production
+  environment-name             = var.environment
+  infrastructure-support       = var.infrastructure_support
+  namespace                    = var.namespace
+  performance_insights_enabled = true
+  db_engine_version            = "13"
+  db_instance_class            = "db.t3.small"
+  rds_family                   = "postgres13"
```

Comment:
again would be better starting with the latest version - think it is 14

Reply:
Our immediate goal is to get an environment in place with as little divergence from the approved premises environment as possible. Can this be allowed for now?